### PR TITLE
Nexus: support Polaris at ALCF

### PR DESCRIPTION
**Pull request by jtkrogel**
## Proposed changes

This PR adds support for Polaris in Nexus.  This includes support for job bundling (which on Polaris requires parsing/splitting/distributing nodelist files).

## What type(s) of changes does this code introduce?

- New feature

### Does this introduce a breaking change?

- No

## What systems has this change been tested on?

Workstation, Polaris

## Checklist

- Yes. This PR is up to date with current the current state of 'develop'

### CODE REVIEW

File: nexus/lib/machines.py [hunk 1]
```diff
@@ -3371,6 +3379,69 @@ def write_job_header(self,job):
...
+            job.run_options.add(**opt)
+        #end if
+    #end def post_process_job
+
+    def write_job_header(self,job):
+        if job.queue is None:
+            job.queue = 'prod'
+        #end if
+        if job.filesystems is None:
+            job.filesystems = 'home:eagle'
```
Comment (markdewing) [inline nexus/lib/machines.py:3405]: Should the default add 'grand' as well?

**Review by prckent — COMMENTED** [review]
What is the quirk that requires specialize_bundle_commands?

File: nexus/lib/machines.py (hunk 1 above)
Comment (jtkrogel) [reply nexus/lib/machines.py:3405]: Sure.  Will add.

Comment (jtkrogel) [reply nexus/lib/machines.py]: Done.

**Comment by prckent** [comment]
ntest_nexus_machines is failing in CI after grand was added. It was fine before.

**Review by prckent — APPROVED** [review]

**Pull request merged**